src/pages/resend/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Button } from '@components/Button'
+import { Container } from '@components/Container'
+import { LeftIcon } from '@components/Icons'
+import Link from 'next/link'
+
+const Resend = () => (
+  <Container className="pb-44">
+    <div className="justify-center pt-5 xl:flex xl:flex-col xl:items-center xl:pt-10">
+      <div className="flex w-full justify-between">
+        <Link href="/">
+          <Button
+            variant="text"
+            leftIcon={<LeftIcon />}
+          >
+            Go back
+          </Button>
+        </Link>
+        <Link href="/login">
+          <Button variant="outlined">Log In</Button>
+        </Link>
+      </div>
+      <div className="mt-[60] xl:mt-10">
+        <h1 className="text-xl text-black">The link was sent to your email</h1>
+        <div className="mt-5 flex items-center border-b border-gray-thin pb-5">
+          <svg
```

Comment:
create custom svg component like we did in other pages

Reply:
done